staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/all_types_match/trivial/testdata/validate-false.json

```diff
@@ -1,6 +1 @@
-{
-    "*trivial.E1": {},
-    "*trivial.E2": {},
-    "*trivial.T1": {},
-    "*trivial.T2": {}
-  }
\ No newline at end of file
+{}
```

Comment:
nit: is this validate-false.json file intended to have an empty struct?

Reply:
This was auto-generated by `UPDATE_VALIDATION_GEN_FIXTURE_DATA=true go test ./staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/...`

Reply:
Ah got it, thanks